src/lib/comms/sol-lwm2m.c

```diff
@@ -3305,7 +3358,8 @@ handle_write(struct sol_lwm2m_client *client,
                     r = write_instance_tlv_or_resource(client, obj_ctx, obj_instance, -1,
                         instance_payload);
                 } else {
-                    r = handle_create(client, obj_ctx, instance_tlvs->id, instance_payload);
+                    r = handle_create(client, obj_ctx, instance_tlvs->id,
+                        instance_payload, UINT16_MAX, false);
```

Comment:
The outermost `if` already has `&& client->is_bootstrapping` as a condition. Should I still add a comment or something in this line?